### PR TITLE
Pin numpy to <1.24 in TensorFlow examples

### PR DESCRIPTION
**Pull request by JanuszL**
- due to a recent release of backward incompatible numpy 2.0
  pin its version in the TensorFlow examples

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- due to a recent release of backward incompatible numpy 2.0
  pin its version in the TensorFlow examples

### Key points relevant for the review:
- NA

### Tests:

- [x] Existing tests apply
  - TL3_YOLO_convergence
  - TL3_EfficientDet_convergence
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Review by jantonguirao — APPROVED** [review]

File: docs/examples/use_cases/tensorflow/efficientdet/requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ argparse-utils
 absl-py
 Pillow
 PyYAML
+numpy<1.24
```
Comment (mzient) [inline docs/examples/use_cases/tensorflow/efficientdet/requirements.txt:6]: Why not `<2`? Also, the description of the PR says "pin to 1.24" whereas it actually pins it to _less than_ 1.24.

**Review by mzient — APPROVED** [review]
Please rename the PR and adjust the commit message to reflect actual contents of the PR. Otherwise OK.

File: docs/examples/use_cases/tensorflow/efficientdet/requirements.txt (hunk 1 above)
Comment (JanuszL) [reply docs/examples/use_cases/tensorflow/efficientdet/requirements.txt:6]: Updated the commit message.

**Pull request merged**